linux/MDATP_Linux_High_CPU_parser.ps1

```diff
@@ -0,0 +1,18 @@
+#Clear the screen
+clear
+# Set the directory path where the output is located
+$Directory = "C:\temp\High_CPU_util_parser_for_Linux"
+# Set the path to where the input file (in Json format) is located
+$InputFilename = ".\real_time_protection_logs"
+# Set the path to where the file (in csv format)is located
+$OutputFilename = ".\real_time_protection_logs_converted.csv"
+# Change directory
+cd $Directory
```

Comment:
It is not obvious who is going to create this directory. I'd just suggest to consume files from the current directory (or, better, from a provided parameter, or maybe even stdin).